investment-app/TRINV.Api/TRINV.Application/Queries/Dashboard/GetUserInvestmentsPerformanceListQuery.cs

```diff
@@ -0,0 +1,98 @@
+﻿namespace TRINV.Application.Queries.Dashboard;
+
+using DigitalCurrency;
+using Domain.Entities;
+using Domain.Enums;
+using Interfaces;
+using MediatR;
+using Shared.Business.Utilities;
+
+public record GetInvestmentPerformanceQuery : IRequest<OperationResult<List<InvestmentPerformanceQueryModel>>>;
```

Comment:
The same here you need to accept Investment Type and use it in currentUserInvestments

Reply:
Done.